client/src/components/ApplicantProfile.js

```diff
@@ -23,23 +23,38 @@ const ApplicantProfile = () => {
   ]);
 
   const [profileCreated, setProfileCreated] = useState(null);
+  const addMoreExperience = () => {
+    setExperienceData([
+      ...experienceData,
+      {
+        company: "",
+        job_title: "",
+        description: "",
+        start_date: "",
+        end_date: ""
+      }
+    ]);
+  };
 
   const handleChange = (event, result) => {
     const { name, value } = result || event.target;
     setProfileData({ ...profileData, [name]: value });
   };
-
-  const handleExperience = ({ event, targetData, experienceIndex }) => {
+  const updateExperienceData = (value, name, index) =>
+    experienceData.map((experience, experienceIndex) => {
+      if (index === experienceIndex) {
+        return { ...experience, [name]: value };
```

Comment:
I think this could be more readably (very arguable and subjective, but it is more concise and will perform slightly better) written as something like

```suggestion
  const updatedExperienceData = [...experiencesData];
  updatedExperienceData[index] = {...updatedExperienceData[index], [name] : value};
  return updatedExperienceData;
}
```

Reply:
this particular change is causing error

Reply:
could you clarify what error it's causing?

Reply:
cant select date with this change